lerna-package-lock.json

```diff
@@ -692,8 +692,8 @@
 		},
 		"@fluidframework/agent-scheduler": {
 			"version": "0.26.10",
-			"resolved": "https://registry.npmjs.org/@fluidframework/agent-scheduler/-/agent-scheduler-0.26.10.tgz",
-			"integrity": "sha512-gT6hnYlqoTwgpS+xy5krNMqBdJu4hVpp/AF0XnrMysLknFJ+iOU/J936owEM9A0SwF4MLS5Q3XjJ8b0juZUtEw==",
+			"resolved": "https://offnet.pkgs.visualstudio.com/officenet/_packaging/fluid/npm/registry/@fluidframework/agent-scheduler/-/agent-scheduler-0.26.10.tgz",
```

Comment:
offnet is you problem

Reply:
user level npmrc that I haven't touched in years.....sigh